Flank/src/main/java/com/walmart/otto/configurator/ConfigReader.java

```diff
@@ -141,6 +136,10 @@ private void setProperty(String property, String value) throws IllegalArgumentEx
         configurator.setGenerateSplitVideo(Boolean.parseBoolean(value));
         break;
 
+      case "skip-tests":
```

Comment:
hehe. I think I was the only one using this feature. I would remove it and mention the change in the changelog.

Reply:
Ok, it makes sense to me.